Samples/Shared/README.md

```diff
@@ -0,0 +1,50 @@
+# Shared Sample Resources
+
+The `Shared` folder contains code, storyboards, property lists, and configuration files that are used by many of the sample applications.
+
+> **IMPORTANT:**
+> The samples include file paths that reference the files in this folder.
+
+There are different types of content:
+
+- **Common:** A folder that contains code and storyboards.
+- **Okta.plist:** A property list that contains the configuration for the Okta org Application Integration that's used by the app.
+- **TestConfiguration.xcconfig:** A file that defines the configuration for the Okta org Application Integration used that's used for testing.
+
+## Common Folder
+
+The contents of the folders are:
+
+- **Keychain:** Convenience extensions to Keychain for tokens.
+- **Testing:** Extensions to XCTest and other classes used for testing the sample apps.
+- **View Controllers:** A view controller that displays the information for a user after they've signed in.
+
+## Okta Property List
+
+Update the values for the keys in the file to configure the connection to your Okta Org Application Integration for your app.
+
+ Key | Required | Description |
+ ---|---|---
+`issuer` | ✔ | The domain of your registered Okta org followed by `/oauth2/default`, such as `https://dev-1234567.okta.com/oauth2/default`.
+`clientId` | ✔ | The client ID from the Application Integration in the Okta Admin console, such as `0ux3rutxocxFX9xyz3t9`.
+`scopes` | ✔ | A space-delimited list of the permissions, or OAuth scopes requested by the client. The existing list may not need updating.
+`redirectUri` | ✔  | Redirect URI for the Okta application, such as `com.okta.1234567:/callback`.
+`logoutRedirectUri` | | Logout URI used for the Okta application, such as `com.okta.1234567:/logout`
+
+Any other keys and values that you add to the plist are passed to the `additionalParameters` argument of the initializer.
+
+> **NOTE:** This file is referenced from the various sample applications included in the OktaMobileSDK workspace, and may be edited directly within the sample projects.
```

Comment:
@mauricesharp-okta I added this line, as well as the companion on line 50, to indicate that the developer doesn't need to directly edit this file, but rather that it's shared among the various sample projects. What are your thoughts?

Reply:
I'd change it slightly to make it clear that editing this file in any of the projects changes the content for all projects. It's a subtle distinction, but one that some developers may not catch. They may wonder why tests in other samples suddenly stop working ;-).

```suggestion
> **NOTE:** You can edit this file in any of the sample app projects. Editing the file in one sample changes it for all samples.
```